tests/robot-tests/tests/admin_and_public/bau/publish_release_and_amend_2.robot

```diff
@@ -202,4 +202,36 @@ Wait for release amendment process status to be Complete
 Go to permalink page & check for error element to be present
     [Tags]  HappyPath
     user goes to url  ${PERMA_LOCATION_URL}
-    user waits until page contains  WARNING - The data used in this permalink may be out-of-date.
\ No newline at end of file
+    user waits until page contains  WARNING - The data used in this permalink may be out-of-date.
+
+# EES-1303
+Return to Admin to start deleting subject
```

Comment:
I'd change this so that you import another subject as part of the original release, and then delete it as part of the amendment, and then check the published amendment and/or table tool doesn't contain the deleted subject. That would provide a much more thorough test.